### PR TITLE
feat(#67): add statistics on links and digests

### PR DESCRIPTION
**Pull request by tlenclos**
Also updated Next.js from 13.5 => 14.

| Team home | Edit digest |
|--------|--------|
| image | image |

### CODE REVIEW

File: src/components/digests/block-card/bookmark-card/card-style/Inline.tsx [hunk 1]
```diff
@@ -43,6 +49,11 @@ function CardStyleInline({
             <p className="text-sm mt-1 text-slate-400">
               {getDomainFromUrl(url)}
             </p>
+            <div className="flex items-center text-sm text-gray-500 mt-1">
+              <div className="flex items-center">
+                <EyeIcon className="text-green-600 h-4 w-4 mr-1" /> {views} views
```
Comment (shinework) [inline src/components/digests/block-card/bookmark-card/card-style/Inline.tsx:54]: view/views

Comment (shinework) [inline src/components/digests/block-card/bookmark-card/card-style/Inline.tsx:54]: not sure about the green color, maybe text-gray-500 is better

**Review by Lucieo — APPROVED** [review]

File: src/components/digests/Digests.tsx [hunk 2]
```diff
@@ -58,6 +58,11 @@ export const Digests = ({ digests, teamSlug }: Props) => {
                 {`${bookmarkCount} bookmark${bookmarkCount > 1 ? 's' : ''}`}
               </div>
             </div>
+            {digest.publishedAt && <div className="flex items-center text-sm text-gray-500">
+              <div className="flex items-center">
+                <EyeIcon className="text-green-600 h-4 w-4 mr-1" /> {digest.views} views
```
Comment (Lucieo) [inline src/components/digests/Digests.tsx:63]: maybe don't display if views is 0

File: src/components/digests/block-card/bookmark-card/card-style/Block.tsx [hunk 3]
```diff
@@ -39,6 +46,11 @@ function CardStyleBlock({
                   fallbackSrc={`${getEnvHost()}/api/bookmark-og?bookmark=${bookmarkId}`}
                 />
               </div>
+              <div className="flex items-center text-sm text-gray-500 mt-1">
+                <div className="flex items-center">
```
Comment (Lucieo) [inline src/components/digests/block-card/bookmark-card/card-style/Block.tsx:50]: don't show for 0 views ?

**Review by quentingrchr — APPROVED** [review]

**Pull request merged**